### ISSUE TITLE
[ONNX] 64-sample Context Window Required for Direct ONNX Usage (not Documented in README)

### ISSUE THREAD

**Issue by OneHoopyFrood** [snakers4/silero-vad#771]
## 🐛 Bug

When using `silero_vad.onnx` directly via `onnxruntime` (Python, C#, JavaScript, mobile, etc.) without the `silero-vad` package wrapper, passing raw 512-sample frames produces near-zero speech probabilities (~0.001) for every frame regardless of audio content — a silent 100% false-negative rate.

The cause is a missing 64-sample audio context window that must be prepended to each 512-sample chunk before inference (effective input shape `[1, 576]`, not `[1, 512]`). The ONNX model's dynamic input shape accepts 512 samples without error, making this failure mode invisible.

## To Reproduce

```python
import onnxruntime
import numpy as np

session = onnxruntime.InferenceSession("silero_vad.onnx")
state = np.zeros((2, 1, 128), dtype=np.float32)

# Load any audio with speech, chunk into 512-sample frames
for chunk in speech_chunks:
    out = session.run(None, {
        "input": chunk.reshape(1, 512),  # ← missing 64-sample context
        "state": state,
        "sr": np.int64(16000),
    })
    print(out[0])  # always ~0.001
```

## Expected behavior

The README's ONNX usage section (or a docstring on `OnnxWrapper`) should note:

> Each 512-sample frame must be prefixed with the last 64 samples from the previous frame before calling inference (effective input `[1, 576]`). Without this context, the model produces near-zero probabilities for all input. Initialize the context to zeros for the first frame.

This is already correctly implemented in `utils_vad.py` (`OnnxWrapper.__call__`) and `examples/cpp/silero-vad-onnx.cpp` (`context_samples = 64`), but is not mentioned anywhere in the README or in any prose documentation.

## Environment

N/A — this is a documentation gap, not a runtime/platform-specific issue. Reproduced with `onnxruntime` in Python and `onnxruntime-react-native` in a React Native app.

## Additional context

- `OnnxWrapper.__call__` in `utils_vad.py`: `x = torch.cat([self._context, x], dim=1)` with `context_size = 64`
- `examples/cpp/silero-vad-onnx.cpp`: `const int context_samples = 64`
- The ONNX model's input tensor is declared with dynamic shape `[None, None]`, so passing `[1, 512]` does not produce an error — it silently returns wrong results
- Introduced/formalized in v5 (release notes mention "some context (part of previous chunk) is passed along with the current chunk" but don't quantify it or document the mechanism for non-Python users)

**Comment by OneHoopyFrood** [comment]
"Bug" is a stretch as this is just a documentation gap, but it took me some significant effort to discover the source of my bug that resulted from not knowing this. Figured that earned it the bug template.

**Comment by snakers4** [comment]
Hi,

This plagued us some time after the release of the new version. 
The issue is that the ONNX-model is actually two models with an if inside.
Hence it has a dynamic shape.

As for the docs - there is no obvious place in the README, and most likely people will keep missing it.
We will check if we can add some warning / exception to ONNX, if it is possible at all.

**Comment by snakers4** [comment]
We checked, it is not possible to raise a warming within ONNX itself.
